set auth-enabled = true,command can't get response

**Describe the bug**
set auth-enabled = true,command can't get response

**To Reproduce**
Steps to reproduce the behavior:
1. vim `/etc/cnosdb/cnosdb.conf` and edit `auth-enabled = true`
2. start cnosdb and cnosdb-cli with config
3. exectute any command

**Expected behavior**
cnosdb-cli output:
`ERR: expected json response, got "text/plain", with status: 500 and response body: "Internal Server Error\n"`
cnosdb output:
```
[2022-05-20T11:01:36.328813+08:00] [ERROR] [http_handler.go:916] ["::1 - - [20/May/2022:11:01:36 +0800] \"POST /query?chunked=true&db=&params=null&q=show HTTP/1.1 \" 200 0 \"-\" \"CnosDBShell/unknown\"  78 [panic:runtime error: invalid memory address or nil pointer dereference] goroutine 153 [running]:\nruntime/debug.Stack()\n\t/usr/local/go/src/runtime/debug/stack.go:24 +0x65\ngithub.com/cnosdb/cnosdb/server.WrapWithRecovery.func1.1()\n\t/home/syk/cnosdb/server/http_handler.go:915 +0xbb\npanic({0xf06c00, 0x18a0c50})\n\t/usr/local/go/src/runtime/panic.go:838 +0x207\ngithub.com/cnosdb/cnosdb/server.WrapWithAuthenticate.func1({0x7f1d8c0e4bc8, 0xc0007080a0}, 0xc000708000?)\n\t/home/syk/cnosdb/server/http_handler.go:1306 +0x6f\nnet/http.HandlerFunc.ServeHTTP(0x11d6ab8?, {0x7f1d8c0e4bc8?, 0xc0007080a0?}, 0x0?)\n\t/usr/local/go/src/net/http/server.go:2084 +0x2f\ngithub.com/cnosdb/cnosdb/server.WrapWithResponseWriter.func1({0x11d6ab8?, 0xc000708080?}, 0x6?)\n\t/home/syk/cnosdb/server/http_handler.go:1464 +0x6e\nnet/http.HandlerFunc.ServeHTTP(0xc00033e090?, {0x11d6ab8?, 0xc000708080?}, 0xffffffffffffff0b?)\n\t/usr/local/go/src/net/http/server.go:2084 +0x2f\ngithub.com/cnosdb/cnosdb/server.WrapWithCors.func1({0x11d6ab8, 0xc000708080}, 0xc0000d6100)\n\t/home/syk/cnosdb/server/http_handler.go:864 +0x694\nnet/http.HandlerFunc.ServeHTTP(0xfada40?, {0x11d6ab8?, 0xc000708080?}, 0xc?)\n\t/usr/local/go/src/net/http/server.go:2084 +0x2f\ngithub.com/cnosdb/cnosdb/server.WrapWithRequestID.func1({0x11d6ab8, 0xc000708080}, 0xc0000d6100)\n\t/home/syk/cnosdb/server/http_handler.go:881 +0x16d\nnet/http.HandlerFunc.ServeHTTP(0x203000?, {0x11d6ab8?, 0xc000708080?}, 0x8?)\n\t/usr/local/go/src/net/http/server.go:2084 +0x2f\ngithub.com/cnosdb/cnosdb/server.(*Handler).WrapWithLogger.func1({0x11d6ab8?, 0xc000708060}, 0x4bd501?)\n\t/home/syk/cnosdb/server/http_handler.go:890 +0xf2\nnet/http.HandlerFunc.ServeHTTP(0x88?, {0x11d6ab8?, 0xc000708060?}, 0xc000183400?)\n\t/usr/local/go/src/net/http/server.go:2084 +0x2f\ngithub.com/cnosdb/cnosdb/server.WrapWithRecovery.func1({0x11da1a8?, 0xc000214460}, 0xc0000d6100)\n\t/home/syk/cnosdb/server/http_handler.go:929 +0x154\nnet/http.HandlerFunc.ServeHTTP(0xc0000d6000?, {0x11da1a8?, 0xc000214460?}, 0x0?)\n\t/usr/local/go/src/net/http/server.go:2084 +0x2f\ngithub.com/gorilla/mux.(*Router).ServeHTTP(0xc000b70240, {0x11da1a8, 0xc000214460}, 0xc00020a600)\n\t/home/syk/go/pkg/mod/github.com/gorilla/mux@v1.8.0/mux.go:210 +0x1cf\ngithub.com/cnosdb/cnosdb/server.(*Handler).ServeHTTP(0xc00014a580, {0x11da1a8, 0xc000214460}, 0x40f0e5?)\n\t/home/syk/cnosdb/server/http_handler.go:249 +0x129\nnet/http.(*ServeMux).ServeHTTP(0xc00004638c?, {0x11da1a8, 0xc000214460}, 0xc00020a600)\n\t/usr/local/go/src/net/http/server.go:2462 +0x149\nnet/http.serverHandler.ServeHTTP({0xc000598b70?}, {0x11da1a8, 0xc000214460}, 0xc00020a600)\n\t/usr/local/go/src/net/http/server.go:2916 +0x43b\nnet/http.(*conn).serve(0xc0001c1900, {0x11db500, 0xc000193290})\n\t/usr/local/go/src/net/http/server.go:1966 +0x5d7\ncreated by net/http.(*Server).Serve\n\t/usr/local/go/src/net/http/server.go:3071 +0x4db\n"] [log_id=0aZrRdIl000]
```

**System information**

Linux 5.16.20-2-MANJARO

--- a/server/http_handler.go
+++ b/server/http_handler.go
@@ -180,6 +180,17 @@ func NewHandler(conf *HTTPConfig) *Handler {
 		accessLogger:   log.New(os.Stderr, "[httpd] ", 0),
 	}
 
+	if metaConfig, err := meta.NewDemoConfig(); err == nil {
+		h.metaClient = meta.NewClient(metaConfig)
+	} else {
+		logger.Error(fmt.Sprintf("unable to create metaclient %s", err))
+	}
+
+	err := h.metaClient.Load()
+	if err != nil {
+		logger.Error(fmt.Sprintf("unable to load metadata %s", err))
+	}
+
 	h.writeThrottler = NewThrottler(conf.MaxConcurrentWriteLimit, conf.MaxEnqueuedWriteLimit)
 	h.writeThrottler.EnqueueTimeout = conf.EnqueuedWriteTimeout
 

